src/valkey_private.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, zhenwei pi <pizhenwei@bytedance.com>
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *   * Redistributions of source code must retain the above copyright notice,
+ *     this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *   * Neither the name of Redis nor the names of its contributors may be used
```

Comment:
We should not create new files with a license mentioning Redis as the name of the project. Let's change this in a follow-up.

A trademark-neutral version of the license mentions "the copyright holder" here instead of Redis.

Reply:
OK. Just use "the copyright holder" instead of "Redis"? Or would you please give a full doc of license example?

Reply:
Will fix this after https://github.com/valkey-io/libvalkey/issues/48 getting ACKed.

Reply:
@michael-grunder created https://github.com/valkey-io/libvalkey/pull/49 to fix this issue, mark this conversation resolved.